table/block_based/block_based_table_reader.cc

```diff
@@ -2866,6 +2992,7 @@ Status BlockBasedTable::Get(const ReadOptions& read_options, const Slice& key,
         rep_->internal_comparator.user_comparator()->timestamp_size();
     bool matched = false;  // if such user key mathced a key in SST
     bool done = false;
+    bool is_referenced_key_exist = false;
```

Comment:
You could limit the scope of this I believe.

Reply:
Thanks. I moved it to where it is used.